### ISSUE TITLE
Automatische Löschung von Benutzerdaten

### ISSUE THREAD

**Issue by mimosel** [tiloman/simren-issues#120]
Unter "Buchungen" lässt sich in simren einstellen, dass Buchungen automatisch nach einer festgelegten Zeit gelöscht werden. Wie ist das aber mit Benutzerdaten? Werden die bei Inaktivität auch automatisch irgendwann gelöscht? Wenn nein, lässt sich das als Option einrichten, dass inaktive Nutzer*innen auch irgendwann gelöscht werden?

**Comment by tiloman** [comment]
Ich werde implementieren dass User nach einer gewissen Zeit Inaktivität eine Info bekommen über die anstehende Löschung. Diese kann dann noch verhindert werden vom User oder wird automatisch durchgeführt.

**Comment by mimosel** [comment]
Die "gewisse Zeit" lässt sich dabei hoffentlich konfigurieren? ;)

**Comment by tiloman** [comment]
Ja, ich würde einen default Wert definieren. Vielleicht 12 Monate nach dem letzten login? Das kann man dann vielleicht noch auf 24 Monate erweitern oder verkürzen auf 6 Monate. Bin für Vorschläge offen.

**Comment by OneOfTwelve** [comment]
Hi, bei uns steht nun an einigen Accounts dran, dass diese bald gelöscht werden.
Dieser Vermerk bleibt auch nach dem erneuten Login erhalten, auch hat der Account keine Info über die bevorstehende Löschung erhalten.

Des weiteren wäre es toll wenn diese Löschung für Accounts mit Teamadmin rechten ausgesetzt ist.

**Comment by tiloman** [comment]
Ja, das ist bislang nur vorbereitet und in progress. Eine löschung findet noch nicht statt aktuell. Der account *müsste* dann morgen wieder den status verlieren wenn heute ein login stattfand. Aber das ist wie gesagt WIP

**Comment by mimosel** [comment]
Für uns wäre dieser issue sehr wichtig. Gibt es hier schon einen zeitlichen Rahmen, den Sie kommunizieren können?

**Comment by tiloman** [comment]
Umgesetzt. Erste Löschungen gibt es in den ersten Teams ab dem 3.3.2025. Betroffene User wurden darüber per Mail informiert (immer 4 Wochen im voraus).
